security/guard_authentication.rst

```diff
@@ -164,14 +164,15 @@ This requires you to implement six methods::
     class TokenAuthenticator extends AbstractGuardAuthenticator
     {
         /**
-         * Called on every request. Return whatever credentials you want,
-         * or null to stop authentication.
+         * Called on every request. Return whatever credentials you want to
+         * be passed to getUser().  Returning null will cause authentication
```

Comment:
It'll not necessary cause the authentication to be successful, given that you can have [multiple guard authenticators](http://symfony.com/doc/current/security/multiple_guard_authenticators.html). It only means this authenticator was unable to extract credentials from the request, and thus give hand to the next ones.